common/src/main/java/org/opensearch/ml/common/transport/connector/MLUpdateConnectorRequest.java

```diff
@@ -57,6 +58,22 @@ public ActionRequestValidationException validate() {
 
         if (updateContent == null) {
             exception = addValidationError("Update connector content can't be null", exception);
+        } else {
+            String modelName = updateContent.getName();
+            String description = updateContent.getDescription();
+
+            if (modelName != null && !isSafeText(modelName)) {
+                exception = addValidationError(
+                    "Model connector name can only contain letters, digits, spaces, underscores (_), hyphens (-), and dots (.). Max length: 1000 characters.",
```

Comment:
Don't see where validate `Max length: 1000 characters."` , Is that built-in logic in `isSafeText`?

Reply:
My bad, I removed that restriction from the regex. But then I forgot to remove that from the error message. 

Initially I was thinking to provide 1000 characters restriction. 

What do you think? Should we add such kind of restriction?

Reply:
Ignore the comment, find that internal logic in `isSafeText`